cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/connector/resource/AwsCloudWatchService.java

```diff
@@ -34,47 +32,46 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(AwsCloudWatchService.class);
 
-    @Value("${freeipa.aws.cloudwatch.suffix:-Status-Check-Failed-System}")
+    @Value("${aws.cloudwatch.suffix:-Status-Check-Failed-System}")
```

Comment:
do we need any helm chart change because of the renamed property?

Reply:
Not really sure what the helm chart even is, hopefully someone else can answer this. These properties are not being injected from anywhere though as they are just being set in place so I'm not sure why anything fancy would need to be done with them.